docs/orchestrating-elastic-stack-applications/stack-helm-chart.asciidoc

```diff
@@ -23,6 +23,11 @@ NOTE: The minimum supported version of Helm is 3.2.0.
 
 Similar to the <<{p}-deploy-elasticsearch,quickstart>>, the following section describes how to setup an Elasticsearch cluster with a simple Kibana instance managed by ECK, and how to customize a deployment using the eck-stack Helm chart's values.
 
+[source,sh,subs="attributes"]
```

Comment:
Why should we only mention the CRDs?

Also that section is right after [Deploy an orchestrator](https://www.elastic.co/docs/deploy-manage/deploy/cloud-on-k8s/deploy-an-orchestrator) which already explain how to install ECK. So I would make that a prerequisite instead.